### PR TITLE
Login store

### PR DESCRIPTION
**Pull request by RazvanMaftei9**

### CODE REVIEW

File: src/app/session/services/auth-guard.service.ts [hunk 1]
```diff
@@ -1,16 +1,24 @@
 import { Injectable } from '@angular/core';
```
Comment (moscugeorge123) [inline src/app/session/services/auth-guard.service.ts]: change this file to be the same as master

File: src/app/session/services/nonauth-guard.service.ts [hunk 2]
```diff
@@ -1,19 +1,34 @@
 import { Injectable } from '@angular/core';
```
Comment (moscugeorge123) [inline src/app/session/services/nonauth-guard.service.ts]: change this file to be the same as master

File: src/app/session/services/session.service.ts [hunk 3]
```diff
@@ -15,13 +15,13 @@ import { environment } from 'src/environments/environment';
   providedIn: 'root',
 })
 export class SessionService {
-  private url: string = environment.url;
+  // private url: string = environment.url;
+  private url: string = "http://localhost:1337";
```
Comment (moscugeorge123) [inline src/app/session/services/session.service.ts]: remove this as you don't need it

File: src/app/session/store/reducers/login.reducers.ts [hunk 4]
```diff
@@ -10,12 +10,14 @@ export const sessionLoginReducers = [
       loading: true
     }
   })),
-  on(SessionLoginSuccess, (state: ISessionState, data: any) => ({
+  on(SessionLoginSuccess, (state: ISessionState, payload: any) => ({
```
Comment (moscugeorge123) [inline src/app/session/store/reducers/login.reducers.ts]: Here shouldn't be `(state: ISessionState, { payload })`?

**Review by moscugeorge123 — CHANGES_REQUESTED** [review]

File: src/app/session/store/reducers/login.reducers.ts (hunk 4 above)
Comment (RazvanMaftei9) [reply src/app/session/store/reducers/login.reducers.ts]: i dont think so, when i tested it it locally it worked just fine (and i dont know how i would access the payload data if i wrapped it in a js object)

**Pull request merged**